src/platform/w3dengine/client/w3ddisplay.cpp

```diff
@@ -65,8 +65,16 @@
 #endif
 #include <stdio.h>
 
+#ifdef GAME_DLL
+#include "hooker.h"
+#endif
+
 #ifdef GAME_DEBUG_STRUCTS
+#ifdef GAME_DLL
 ICoord2D &s_leftClickReleased = Make_Global<ICoord2D>(0x00E1B118);
```

Comment:
Is it supposed to be `ICoord2D` or `ICoord2D *` ?

Reply:
Its supposed to be ICoord2D.